topdown/trace.go

```diff
@@ -181,3 +185,25 @@ func (ds depths) GetOrSet(qid uint64, pqid uint64) int {
 	}
 	return depth
 }
+
+func builtinTrace(bctx BuiltinContext, args []*ast.Term, iter func(*ast.Term) error) error {
+
+	str, err := builtins.StringOperand(args[0].Value, 1)
+	if err != nil {
+		return handleBuiltinErr(ast.Trace.Name, bctx.Location, err)
+	}
+
+	evt := &Event{
+		Op:       NoteOp,
+		Node:     str,
```

Comment:
We should use a new field called Message here.

There are two places that need to be updated to support a new field.

* formatEvent in this file (https://github.com/jyoverma/opa/blob/5c4afe76d6434c7eefc93cafd00c2908728129f4/topdown/trace.go#L148)
* TraceEventV1 in the server/types package: https://github.com/open-policy-agent/opa/blob/master/server/types/types.go#L241